api/src/pcapi/connectors/boost.py

```diff
@@ -20,15 +20,20 @@
 
 class ResourceBoost(enum.Enum):
     EXAMPLE = "example"
+    EXAMPLE_WITH_PATTERNS = "example/{start}/{end}"
     FILMS = "films"
     SHOWTIMES = "showtimes"
+    SHOWTIME = "showtimes/{id}"
```

Comment:
Je profite de cette PR pour ajouter `api/` au début des endpoint, ou sinon l'enlever pour l'endpoint du login et l'incluras dans les URLs à la db
```suggestion
    EXAMPLE = "api/example"
    EXAMPLE_WITH_PATTERNS = "api/example/{start}/{end}"
    FILMS = "api/films"
    SHOWTIMES = "api/showtimes"
    SHOWTIME = "api/showtimes/{id}"
```

Reply:
Ajoutons `api/` , bien vu !